nservicebus/databus.md

```diff
@@ -0,0 +1,69 @@
+---
+title: DataBus feature
+summary: DataBus feature
+tags:
+- DataBus
+- Attachments
+---
+
+## What is DataBus feature for?
+
+Messages are intended to be small. Some scenarios require to send large binary data along with a message. For this purpose, NServiceBus has a DataBus feature to allow to overcome limitations of message size.

```

Comment:
I'd add that the limitation is imposed by the underlying transport and to be sure to be able to easily switch transport or to accomplish new requirements each time we try to move something that **can** be big it should be transported with the DataBus